src/calculateRentalCost.js

```diff
@@ -6,5 +1,22 @@
+const LONG_TERM = 3;
+const BASEPRISE = 40;
+const SELL = 20;
+
+const GOOD_DAYS = 7;
+const DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const fullCost = days * BASEPRISE;
+
+  if (days >= GOOD_DAYS) {
+    return fullCost - DISCOUNT;
```

Comment:
The discount logic here assumes a flat discount of 50 for rentals of 7 days or more. Ensure this is the intended logic, as it might be more common to apply a percentage discount or a different fixed amount.